src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2796,6 +2796,10 @@ static bool createEntity
       {
         oeP->fill(SccInvalidModification, "Already Exists", "Unprocessable");
       }
+      else if (errDetail->find("Can't extract geo keys") != std::string::npos)
+      {
+        oeP->fill(SccBadRequest, "Bad Request", "BadRequest");
```

Comment:
In order to provide a more informative feedback for user I'd suggest:

```suggestion
        oeP->fill(SccBadRequest, "Wrong GeoJson", "BadRequest");
```

Reply:
(.test file would need to be adapted accordingly)

Reply:
Fixed in ae914b6